polyfactory/factories/base.py

```diff
@@ -1068,9 +1069,25 @@ def process_kwargs_coverage(cls, **kwargs: Any) -> abc.Iterable[dict[str, Any]]:
                 resolved[field_name] = post_generator.to_value(field_name, resolved)
             yield resolved
 
+    @classmethod
+    async def build_async(cls, **kwargs: Any) -> T:
+        """Asynchronously build an instance of the factory's __model__
+
+        :param kwargs: Any kwargs. If field names are set in kwargs, their values will be used.
+
+        :returns: An instance of type T.
+
+        """
+        data: dict[str, Any] = cls.process_kwargs(**kwargs)
+        for k, v in data.items():
+            if isinstance(v, Coroutine):
```

Comment:
Is this behaviour generally used for other async methods or more specific to SQLA relationships? If specific to SQLA then should live within that factory

Reply:
I should check a bit, but could suppose this is more specific for SQLAlchemy proxy. If this behaviour is only for the Alchemy, I'll try to think something and leave it within SQLAlchemy factory.